cornflow/endpoints/case.py

```diff
@@ -113,6 +113,7 @@ def get_execution_data(execution_id):
                 raise ObjectDoesNotExist("Execution does not exist")
             data = get_instance_data(execution.instance_id)
```

Comment:
I'm not sure about this but shouldn't you update the `get_instance_data()` function so that it also returns the instance checks ?

Reply:
You are right. Changed